fsw/src/bpnode_dispatch.c

```diff
@@ -0,0 +1,135 @@
+/************************************************************************
+ * NASA Docket No. GSC-18,719-1, and identified as “core Flight System: Bootes”
+ *
+ * Copyright (c) 2020 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain
+ * a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ************************************************************************/
+
+/**
+ * \file
+ *   This file contains the source code for the BPNode software bus processing.
+ */
+
+/*
+** Include Files:
+*/
+#include "bpnode_app.h"
+#include "bpnode_dispatch.h"
+#include "bpnode_cmds.h"
+#include "bpnode_eventids.h"
+#include "bpnode_msgids.h"
+#include "bpnode_msg.h"
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * **/
+/*                                                                            */
+/* Verify command packet length                                               */
+/*                                                                            */
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * **/
+bool BPNode_VerifyCmdLength(const CFE_MSG_Message_t *MsgPtr, size_t ExpectedLength)
+{
+    bool              result       = true;
```

Comment:
```suggestion
    bool              Result       = true;
```

Reply:
Yes, I feel a little bad for posting such a benign comment, but also its the only variable that doesn't match. D':

Reply:
Hey as long as you don't say anything when I post similarly minor comments on other MRs, I'll take it

Reply:
I know lol

Reply:
Done